protocol/communities/community.go

```diff
@@ -902,6 +902,7 @@ func (o *Community) Leave() {
 
 func (o *Community) Spectate() {
 	o.config.Spectated = true
+	o.config.Joined = false // this doesn't seem to fix anything, but looks proper to do
```

Comment:
This is probably not needed. It defaults to false. We can keep it or not.

Reply:
Reverted.